internal/k8s/controller.go

```diff
@@ -190,6 +190,29 @@ func AddRoutesFromNSToIngestionQueue(numWorkers uint32, c *AviController, namesp
 		utils.AviLog.Debugf("key: %s, msg: %s for namespace: %s", key, msg, namespace)
 	}
 
+}
+func AddServicesFromNSToIngestionQueue(numWorkers uint32, c *AviController, namespace string, msg string) {
+	// For Advancd L4 and service api , do not handle. services already been taken care
+	// in service handler
+	if lib.GetAdvancedL4() || lib.UseServicesAPI() {
+		return
+	}
+	svcObjs, err := utils.GetInformers().ServiceInformer.Lister().Services(namespace).List(labels.Set(nil).AsSelector())
+	if err != nil {
+		utils.AviLog.Errorf("Unable to retrieve the services during full sync: %s", err)
+		return
+	}
+	for _, svcObj := range svcObjs {
+		isSvcLb := isServiceLBType(svcObj)
+
+		if isSvcLb {
+			key := utils.L4LBService + "/" + utils.ObjKey(svcObj)
+			bkt := utils.Bkt(namespace, numWorkers)
+			c.workqueue[bkt].AddRateLimited(key)
+			utils.AviLog.Debugf("key: %s, msg: %s for namespace: %s", key, msg, namespace)
```

Comment:
here also - we might just handles "Services" in general as opposed to only service of type "Loadbalancer"